rmw_fastrtps_shared_cpp/src/rmw_event.cpp

```diff
@@ -164,4 +164,9 @@ __rmw_event_set_callback(
   return RMW_RET_OK;
 }
 
+bool rmw_event_check_compatible(rmw_event_type_t rmw_event_type)
```

Comment:
I would follow the same pattern we use with other methods:

```suggestion
bool __rmw_event_check_compatible(rmw_event_type_t rmw_event_type)
```

Reply:
And then call this method from `rmw_fastrtps_cpp` and `rmw_fastrtps_dynamic_cpp`